### PR TITLE
HG-838 Don't display deprecations when running unit tests

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/HG-838

/cc @hakubo @lizlux @mari-mcmurtrie

### CODE REVIEW

File: test/helpers/startEmberApp.js [hunk 1]
```diff
@@ -32,3 +32,6 @@ App.initializer({
 setResolver(Ember.DefaultResolver.create({
 	namespace: App
 }));
+
+// Don't clutter test log with deprecations
+Ember.deprecate = function(){};
```
Comment (hakubo) [inline test/helpers/startEmberApp.js]: Em.K :)

**Comment by hakubo** [comment]
LGTM

**Comment by lizlux** [comment]
LGTM. Could probably use a more descriptive comment about what `Em.deprecate` and `Em.K` are (I had to look both of them up).

**Comment by rogatty** [comment]
@lizlux well... Should we add explanatory comments for all Ember mechanics or only for some subset? ;) IMO that's how it goes when you work with a framework - you need to check documentation for classes or functions you haven't seen before. I don't think that we should copy and paste docs into comments, only write about things that are not obvious either from the context or Ember documentation.

**Comment by lizlux** [comment]
You have a good point, and I'm fine with leaving it as is, but this could be more descriptive and more helpful to someone looking at it quickly: "Set deprecation warning method to Ember's version of noop to declutter test logs".

**Comment by rogatty** [comment]
@lizlux: clarified in the comment, thanks for suggestion.

**Comment by lizlux** [comment]
:+1:

**Pull request merged**